mplc/partner.py

```diff
@@ -32,12 +32,44 @@ def __init__(self, partner_id):
         self.y_val = None
         self.y_test = None
 
-    def corrupt_labels(self):
-        for label in self.y_train:
+    def corrupt_labels(self, proportion_corrupted):
+        if proportion_corrupted>1 or proportion_corrupted<0:
```

Comment:
I think this could be easier to read as `if not 0 < proportion_corrupted < 1:`.
And you could also use f-strings for the exception error.
But this is only a suggestion